### PR TITLE
[BE-1] User model created and database changed to PostgreSQL.

### PR DESCRIPTION
**Pull request by Ridvansezen**
@AnilKOC

### CODE REVIEW

File: hospital_management_system/user/models.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
 from django.db import models
 
-# Create your models here.
+class User(models.Model):
```
Comment (AnilKOC) [inline hospital_management_system/user/models.py]: AbstractUser dan inherit alalim gelecekde. Django nun yapisini olabildigince kullanalim.

**Review by AnilKOC — APPROVED** [review]

**Comment by Ridvansezen** [comment]
Changed the User model to AbstractUser and made some changes.

@AnilKOC

**Review by AnilKOC — APPROVED** [review]

**Comment by AnilKOC** [comment]
Approved, can you give me access for the repository?

**Comment by Ridvansezen** [comment]
> Approved, can you give me access for the repository?

Access to the repository has been granted. 

@AnilKOC

**Pull request closed without merging**